scripts/publish.sh

```diff
@@ -28,5 +28,28 @@ echo "$NPM_REGISTRY_NO_PROTO/:_authToken=\"$NPM_TOKEN\"" > $NPMRC_PATH
 
 # cd to the package and publish it!
 PKG_PATH=`yarn workspaces info -s | jq -r ".[\"$PKG_NAME\"].location"`
-cd $PKG_PATH
+pushd $PKG_PATH > /dev/null
 npm publish --registry $NPM_REGISTRY
+popd > /dev/null
+
+# === BEGIN Boilerplate Rebuild ===
+
+if [[ "$PKG_NAME" == "zapier-platform-core" ]]; then
+    LEGACY_VERSION=$(curl https://registry.npmjs.org/zapier-platform-legacy-scripting-runner/latest | jq -r .version)
+    CORE_VERSION=$(cat $PKG_PATH/package.json | jq -r .version)
+fi
+
+if [[ "$PKG_NAME" == "zapier-platform-legacy-scripting-runner" ]]; then
+    CORE_VERSION=$(curl https://registry.npmjs.org/zapier-platform-core/latest | jq -r .version)
+    LEGACY_VERSION=$(cat $PKG_PATH/package.json | jq -r .version)
+fi
```

Comment:
So this means we're going to update the boilerplate when either `core` or `legacy-scripting-runner` has a new version.

- If `core` has a new version, we build a new boilerplate with the new `core` and the existing latest `legacy-scripting-runner`.
- If `legacy-scripting-runner` has a new version, we build a boilerplate with the existing latest `core` and the new `legacy-scripting-runner`. Then we replace the boilerplate with the new one we just built.